### PR TITLE
Remove redundant pruning statistics. Align statistics report

### PR DESCRIPTION
**Pull request by evgeniya-egupova**
Two main changes:
1. Some statistics are removed from printing to console. 
from 
pruning_stats
to 
statistics new

2. Filter pruning level in PyTorch is now calculated using a layer mask.

\+ added to TF statistics printing after initialization

### CODE REVIEW

**Comment by evgeniya-egupova** [comment]
Jenkins please retry a build

**Comment by alexsu52** [comment]
@evgeniya-egupova Could you provide before/after comparision?

**Review by vshampor — APPROVED** [review]

File: examples/tensorflow/classification/main.py [hunk 1]
```diff
@@ -229,6 +229,9 @@ def run(config):
     }
 
     if 'train' in config.mode:
+        statistics = compression_ctrl.statistics()
+        logger.info(statistics.to_str())
+
```
Comment (alexsu52) [inline examples/tensorflow/classification/main.py:234]: Could you log statistics using Compression_callbacks for the training step? Explicitly logging statistics before evaluation is good for me and doesn't require changes in this PR.

**Review by alexsu52 — CHANGES_REQUESTED** [review]

**Review by alexsu52 — APPROVED** [review]

**Comment by alexsu52** [comment]
Jenkins please retry a build

**Comment by evgeniya-egupova** [comment]
Jenkins please retry a build
